extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsBuildTimeConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.kafka.streams.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "kafka-streams", phase = ConfigPhase.BUILD_TIME)
+public class KafkaStreamsBuildTimeConfig {
+
+    /**
+     * Whether or not an healtcheck is published in case the smallrye-health extension is present (default to true).
```

Comment:
Minor nit: please add a space between health and check.
Another PR will add it on the existing Javadoc better have all those aligned